public/index.html

```diff
@@ -18,13 +18,13 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div
       id="root"
-      data-wheelPath="/bin/dist/cin_validator-0.1.0-py3-none-any.whl"
-      data-tool="ToolCIN"     
+      data-wheelPath="/bin/dist/lac_validator-2.0.0-py3-none-any.whl"
+      data-tool="Tool903"    
```

Comment:
Why are we changing to 903? I believe you told me before, but I forgot, sorry. Just double checking this is the intended behaviour, because it looks strange

Reply:
We change the data-tool and data-wheelPath parameters to point to the tool whose frontend we want to run.
I changed this so that the frontend ran the 903 tool's wheel file instead and I could check that everything works as expected. 

I should change it back. Good shout.
The default should be CIN.